line-icons/LineIcons.css

```diff
@@ -6,8 +6,12 @@ Author: lineicons.com
 -------------------------------- */
 @font-face {
   font-family: 'LineIcons';
-  src: url('../fonts/LineIcons.eot');
-  src: url('../fonts/LineIcons.eot') format('embedded-opentype'), url('../fonts/LineIcons.woff2') format('woff2'), url('../fonts/LineIcons.woff') format('woff'), url('../fonts/LineIcons.ttf') format('truetype'), url('../fonts/LineIcons.svg') format('svg');
+  src: url('./fonts/LineIcons.eot?y2l643');
```

Comment:
How will this build if fonts are one folder above and not in adjacent file path.

try `npm run build` locally to get the issue

Reply:
just re-ran everything locally as well as installing it directly from my fork. It builds and renders perfectly, including the demo page. This should be good to merge.